### ISSUE TITLE
Gauge does not render on iPad, iPhone, or IE8 under Vista

### ISSUE THREAD

**Issue by orwant** [google/google-visualization-issues#463]
```
What steps will reproduce the problem? Please provide a link to a
demonstration page if at all possible, or attach code.
1. http://code.google.com/apis/visualization/documentation/gallery/gauge.html
2.
3.


What component is this issue related to (PieChart, LineChart, DataTable,
Query, etc)? only Gauge component, everything else works great. The data is presented
overwriting itself, no gauge drawing at all.


Are you using the test environment (version 1.1)?
(If you are not sure, answer NO) NO


What operating system and browser are you using? iPad IOS 4.2.1, iPhone IOS 4.1, or
IE8 (under Vista)




*********************************************************
For developers viewing this issue: please click the 'star' icon to be
notified of future changes, and to let us know how many of you are
interested in seeing it resolved.
*********************************************************

```

Original issue reported on code.google.com by `marc.tarpenning` on 2010-12-03 00:23:06

**Comment by orwant** [comment]
```
There's a working lib that works on iPads here  http://code.google.com/p/jsgauge/

It's a little quirky, but for the most basic needs will work.

My question is why isn't the GVIZ team looking at this code for use on iPad and similar
devices?
```

Original issue reported on code.google.com by `nabeel.azar` on 2011-07-25 13:54:22

**Comment by orwant** [comment]
```
We are working on solving the Gauge issues with iOS and IE8. Will update the bug when
this is resolved.
```

Original issue reported on code.google.com by `amitw@google.com` on 2011-08-02 14:42:57

**Comment by orwant** [comment]
```
Can you please verify that the issue was fixed?
```

Original issue reported on code.google.com by `yinnon@google.com` on 2011-08-03 19:18:26

**Comment by orwant** [comment]
```
The fix is now (finally) live in 1.0.
```

Original issue reported on code.google.com by `jinji.viz` on 2011-08-21 11:10:05
